### DISCUSSION TITLE
I want to know how can i install fast dds using binary in linux systems

### DISCUSSION THREAD

**Discussion by Akul1903** [eProsima/Fast-RTPS-docs#897] [General]
hello, actually i am trying to install fast dds in linux system using binary, but the issue is that the download link of the repository is not working kindly help me out

**Comment by JesusPoderoso** [comment]
Hi @Akul1903, thanks for using Fast DDS.
Please, refer to our documentation webpage, particularly the [Linux installation from binaries](https://fast-dds.docs.eprosima.com/en/latest/installation/binaries/binaries_linux.html) section.

**Reply by Akul1903** [reply]
Hi @JesusPoderoso thanks for your words

when i tried to visit this page the download link is not working
this is the particular link : "https://eprosima.com/index.php/downloads-all"

and without downloading reposository, i cannot install fast dds

kindly provide some solution how to acess this link

**Reply by Akul1903** [reply]
hi @JesusPoderoso 
Can you help me
it's really needful

**Reply by JesusPoderoso** [reply]
Hi @Akul1903, the webpage is being upgraded and may not be available in some countries. Sorry for the inconvenience.
Can the [Linux installation from sources](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html) section suit your needs?

**Reply by Akul1903** [reply]
Thanks

**Reply by Akul1903** [reply]
Hi @JesusPoderoso 
I have installed fast dds on linux the latest version v2.14.2 which has been installed, but after installing there was no setup.bash file was formed.
and without that file we are not able to achive client server architecture.

is there any new command for latest version

**Comment by JesusPoderoso** [comment]
Closing this thread in favor of 
- https://github.com/eProsima/Fast-DDS/discussions/5172

**Reply by Akul1903** [reply]
thanks
